package.json

```diff
@@ -23,50 +23,50 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
-    "bluebird": "2.2.2",
-    "body-parser": "1.12.2",
+    "bluebird": "2.9.34",
+    "body-parser": "1.13.2",
     "bower": "1.4.1",
     "connect-cachify": "0.0.17",
     "connect-fonts-clearsans": "0.0.1",
     "connect-fonts-firasans": "0.0.4",
-    "consolidate": "0.10.0",
-    "convict": "0.8.0",
-    "cookie-parser": "1.3.4",
-    "cors": "2.6.0",
-    "express": "4.12.3",
+    "consolidate": "0.13.1",
+    "convict": "0.8.2",
```

Comment:
Not sure if we try and merge https://github.com/mozilla/node-convict/pull/85 PR and then pin it to `1.0.0`.
/cc @zaach